Unittest flavor list

ajiang38740 opened the pull request:
Add unit test for flavor list.

bhaktadn committed:
Removing previous unit tests folder with duplicate identity config file

Change-Id: Ic8f84dd113787f58214ee29d768e1bfe970625cb

bhaktadn committed:
Adding unit tests for flavor lists.

Change-Id: Ic8ed3ac8da657dfd9168b6976d96916f10dc8437

sudorandom commented on tests/jumpgate-tests/compute/test_flavors.py:
```diff
@@ -0,0 +1,111 @@
+        '''Testing the flavor-list without any parameters set for filtering'''
+        self.param = 'no_params'
+        self.perform_flavor_list(TENANT_ID, 1)
+        self.assertEquals(self.resp.body.keys(), ['flavors'])
```
In python 3, a lot of things return generators instead of lists. You probably need to wrap self.resp.body.keys() with list(self.resp.body.keys) to get the travis tests to pass.

ajiang38740 replied:
Thanks Kevin. I just found out dict.keys() doesn't return list() in python 3.3 any more.

Commit:
1. Fix flavor list unit test python 3.3 compatibility issue.
    2. tox.ini: setup pep8 ignore and exclude list.
    3. add pylintrc to customize pylint

Change-Id: I43a1e0c3243cc8c6b3aa1e1a7d43c837cc5fe7a8

sudorandom commented:
The library I was alluding to earlier is this: https://github.com/openstack-dev/hacking

A lot of the ignores in nova refer to codes defined with this. There are many of them that they refer to as TODOs: https://github.com/openstack/nova/blob/master/tox.ini#L59. If we pass them, I don't think we should include those in the list.

That said, this can be an effort for another pull request.
